packages/extension-bold/package.json

```diff
@@ -29,7 +29,10 @@
     "dist"
   ],
   "devDependencies": {
-    "@tiptap/core": "^2.11.5"
+    "@tiptap/core": "^2.11.5",
+    "@types/jsdom": "^21.1.7",
+    "jsdom": "^26.0.0",
+    "vitest": "^3.0.5"
```

Comment:
I'd not make this a dev dep of the package. It should only be for testing & having it in the root should be enough